doc/ref_arch/openstack/chapters/chapter01.rst

```diff
@@ -147,10 +147,51 @@ to address them (for example, resources autoscaling).
 Terminology
 -----------
 
-General terminology definitions can be found in
-:doc:`cntt:common/glossary` and specific terms relating
-to this reference architecture are to be found in `OpenStack Related
-Terminology :ref:`cntt:common/glossary:openstack related terminology`.
+.. glossary::
+
+    Cloud Infrastructure
+      A generic term covering **NFVI**, **IaaS** and **CaaS** capabilities -
+      essentially the infrastructure on which a **Workload** can be executed.
+
+.. note::
```

Comment:
Should this "note" be prior to the term "Cloud Infrastructure"?

Reply:
@pgoyal01 I did 2 changes in 1 here (we should never do that). note is a sphinx directive which out of glossary. It highlight as such a previous "note:"
https://sublime-and-sphinx-guide.readthedocs.io/en/latest/notes_warnings.html